src/js/client/canHandleStrings.js

```diff
@@ -0,0 +1,57 @@
+// Convenience grade to deal with APIs (like express-couchUser) that return JSON as a string.  The only expected use
+// of this is in combination with an `ajaxCapable` grade, which has the rest of the wiring that uses the options and
+// invokers outlined here.  Instantiating this grade by itself will not work.
+//
+/* global fluid */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.express.user.frontend.canHandleStrings");
+
+    gpii.express.user.frontend.canHandleStrings.stringToJSON = function (that, data) {
+        if (typeof data === "string") {
```

Comment:
As seen in previous contexts, this "convert to JSON if you can but otherwise do nothing" is a suspicious architectural smell - it implies that the workflow is not clear on what type of data it is expecting, and will lead to problems in the future.